### ISSUE TITLE
0crat is not managing long PR reviews

### ISSUE THREAD

**Issue by fanifieiev** [zerocracy/farm#2214]
I contribute to [Cactoos](https://github.com/yegor256/cactoos) and [Takes ](https://github.com/yegor256/takes)projects.
One of my concerns is that some reviewers do not complete their review of PR assigned to them.
One of my PRs(https://github.com/yegor256/takes/pull/1001) has been in review status for approx 40 days and has been looked into by reviewer only once. 
Any attempts to involve the architect(by dropping comments in the PR) of the project lead to no success to have the PR reviewed.

What I would like to have from `0crat` is to manage that kind of situation, for instance, reassign PR to a new reviewer after a certain period(the same as for DEV) and get penalty to the reviewer(if really long). 
The problem is that I can only have max of 3 tasks at the moment, but having delayed reviews, blocks me from being assigned to new issues and that upsets me.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@fanifieiev[/z](https://www.0crat.com/u/fanifieiev) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by g4s8** [comment]
@fanifieiev it's a duplicate of #2196

**Comment by 0crat** [comment]
Job `gh:zerocracy/farm#2214` is not assigned, can't get performer

**Comment by fanifieiev** [comment]
> @fanifieiev it's a duplicate of #2196

@g4s8 Thanks, I have read #2196 . Based on your answer there is no way to reassign a PR if reviewed for long.
That is true, the REV cannot control it, when PR submitter delays CR fixes, but what about the case when PR submitter submitted changes and is waiting for another review for really long time????

As far as I understand the time period spent for REV since the latest commit can be evaluated as:
`now - time_of_latest_commit`.  
So based on the above formula, `0crat` could manage how long the REV spent for the review.

**Comment by g4s8** [comment]
@fanifieiev such cases should be handled by project architect, so you need to ping him or maybe ping PO to take some actions.
